public/app/features/dashboard-scene/serialization/DashboardSceneSerializer.ts

```diff
@@ -49,7 +50,7 @@ export interface DashboardSceneSerializerLike<T, M, I = T, E = T | { error: unkn
     }
   ) => DashboardChangeInfo;
   onSaveComplete(saveModel: T, result: SaveDashboardResponseDTO): void;
-  getTrackingInformation: (s: DashboardScene) => DashboardTrackingInfo | undefined;
+  getTrackingInformation: (s: DashboardScene) => DashboardTrackingInfo | DashboardV2TrackingInfo | undefined;
```

Comment:
```suggestion
  getTrackingInformation: (s: DashboardScene) => (DashboardTrackingInfo & Partial<DashboardV2TrackingInfo>) | undefined;
```

I think this might be a better type. Not tested, just a gut feeling.